docs/2.0/docs/pipelines/guides/updating-pipelines.md

```
@@ -23,6 +23,25 @@ Due to our integration with [Dependabot](https://docs.github.com/en/code-securit
 Gruntwork recommends leaving your workflow reference at a major-tag shorthand, such as `v3` to automatically get the latest minor-tag updates.
 :::
 
+### Pipelines Release Notifications
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)